lifecycle-operator/controllers/common/config/config.go

```diff
@@ -15,20 +15,26 @@ type IConfig interface {
 	GetCloudEventsEndpoint() string
 	SetDefaultNamespace(namespace string)
 	GetDefaultNamespace() string
+	SetBlockDeployment(value bool)
+	GetBlockDeployment() bool
 }
 
 type ControllerConfig struct {
 	keptnAppCreationRequestTimeout time.Duration
 	cloudEventsEndpoint            string
 	defaultNamespace               string
+	blockDeployment                bool
 }
 
 var instance *ControllerConfig
 var once = sync.Once{}
 
 func Instance() *ControllerConfig {
 	once.Do(func() {
-		instance = &ControllerConfig{keptnAppCreationRequestTimeout: defaultKeptnAppCreationRequestTimeout}
+		instance = &ControllerConfig{
+			keptnAppCreationRequestTimeout: defaultKeptnAppCreationRequestTimeout,
+			blockDeployment:                true,
```

Comment:
didn't we say this is disabled by default? I do not recall

Reply:
blockDeployment == true means that the deployment will be blocked if the checks fail (so basically the current behavior we have now), setting it to false won't block the deployment

Reply:
yes but I tought the idea of adding this feature was to not block deployments by default so that people can try out the feature in their environments

Reply:
without worring of pending forever pods

Reply:
but ok the ticket says true by default so 🤷‍♀️